### DISCUSSION TITLE
Season pack snatch picking provider with search delay

### DISCUSSION THREAD

**Discussion by Rouzax** [pymedusa/Medusa#7492] [Ideas & Feature requests]
I have multiple private torrent providers and rarbg with a 24 hour search delay, all providers are setup to perfer season pack and fallback.

The Marvelous Mrs. Maisel is setup as 1080p WebDL and prefered 2160p
![image](https://user-images.githubusercontent.com/4103090/70861422-ef7a9900-1f2d-11ea-9f53-60d87c975ba2.png)

I already had the 1080p and now it downloaded from rarbg while other private indexers also had it.

It looks like the delay is only working after initial broadcast and starts counting then. It would be nice if it also downrated the indexer afterwards if multiple results are returned

**Screenshots**
![image](https://user-images.githubusercontent.com/4103090/70861404-a32f5900-1f2d-11ea-8fdd-a31af2bad913.png)

Medusa Info: | Branch: masterCommit: d0c136d7a528a471b51676140bd35d24d97f65c6Version: 0.3.9Database: 44.14
-- | --
Python Version: | 3.7.4 (tags/v3.7.4:e09359112e, Jul 8 2019, 20:34:20) [MSC v.1916 64 bit (AMD64)]
SSL Version: | OpenSSL 1.1.1c 28 May 2019
OS: | Windows-10-10.0.17763-SP0
Locale: | nl_NL.cp1252

**Debug logs (at least 50 lines):**
General > Advanced Settings > Enable debug
[medusa_log.txt](https://github.com/pymedusa/Medusa/files/3964803/medusa_log.txt)

**Comment by medariox** [comment]
If you sort the providers in the order you prefer (e.g. rarbg last), it should pick the result in the sorted order (if they have the same score).

**Comment by Rouzax** [comment]
I know but I actually want all my Private trackers preferred equally.
Maybe an idea to work with groups?
That way I can set Private trackers in Group 1 (which has priority) and treat all equal in that Group and then set public trackers in Group 2.
Then if delay settings could be set on groups that would also help.
Maybe even extend the groups between NZB and Torrent
